inc/utils.h

```diff
@@ -65,4 +89,29 @@ bool	ft_contains(float x, t_interval interval);
  */
 bool	ft_surrounds(float x, t_interval interval);
 
+/**
+ * @brief Returns the bigger of two values
+ * 
+ * @param x 	First value
+ * @param y 	Second value
+ * @return float 
+ */
+float	ft_max(float x, float y);
```

Comment:
wasn't aware of it, replaced now

Reply:
I heard that the point of math.h is precision, which is rather slow. Since we don't really care about that, we could also use a self made version.
I don't know if it really matters with max() though

Reply:
I would leave it as is. Don't think it will have any effect on our little prog